Examples/KlaviyoSwiftExamples/CocoapodsExample/Podfile.lock

```diff
@@ -1,10 +1,10 @@
 PODS:
   - AnyCodable-FlightSchool (0.6.7)
-  - KlaviyoCore (0.1.0):
+  - KlaviyoCore (4.0.0):
```

Comment:
Do we want to recommend people include klaviyocore, it's not a transitive depedency?

Reply:
No folks don't need to worry about it, it is a transitive dependency. By pulling in `KlaviyoSwift` you are also pulling in `KlaviyoCore` without having to explicitly mention it in your pod file. Updated the example app from demo [here](https://github.com/klaviyo/klaviyo-swift-sdk/blob/master/Examples/KlaviyoSwiftExamples/CocoapodsExample/Podfile).

Reply:
oh my bad this is the lock!

Reply:
Si.